### PR TITLE
Community call readme revision

### PR DESCRIPTION
**Pull request by GBKS**
- Added info for recent use of Twitter Spaces
- Added list of things to consider when hosting calls
- Minor updates

Anything else we should add?

### CODE REVIEW

File: Community-Calls.md [hunk 1]
```diff
@@ -30,28 +30,42 @@ Some examples:
 **How are calls announced?**
 
 - An issue is created 1-2 months earlier
-- They appear on the [calendar](README.md#the-bitcoin-design-calendar)
+- They appear on the [calendar](https://bitcoin.design/contribute/#the-bitcoin-design-calendar)
 - Two @channel message are sent out on Slack, the first one 1 week ahead, and the second one 1 day ahead of the call
-- A regular Slack in the #general channel is sent out shortly before the call
+- A regular Slack post in the #general channel is sent out shortly before the call
+- For Twitter Spaces, the Space is shared via a Tweet when it starts
```
Comment (Bosch-0) [inline Community-Calls.md:36]: You can schedule the Twitter space on twitter so people can subscribe and get notified via Twitter when its about to start also.

File: Community-Calls.md [hunk 2]
```diff
@@ -30,28 +30,42 @@ Some examples:
...
 
 **What tools do we use?**
 
-We use [Jitsi](https://jitsi.org), which is open-source, doesn't require you to create an account, and is available as web and desktop applications.
+For most calls, we have used [Jitsi](https://jitsi.org), which is open-source, doesn't require you to create an account, and is available as web and desktop applications.
```
Comment (danielnordh) [inline Community-Calls.md:40]:
```suggestion
For most calls, we have used [Jitsi](https://jitsi.org), which is open-source, doesn't require you to create an account, and is available as web, mobile and desktop applications.
```

**Review by danielnordh — DISMISSED** [review]
LGTM with the addition of pre-announcing the Twitter space so people can subscribe/get notified.

**Comment by GBKS** [comment]
@danielnordh thanks for the review. I just added a note about scheduling spaces. Could you do another approving review, please?

**Review by danielnordh — APPROVED** [review]

**Pull request merged**